notebooks/03_visualize.ipynb

```diff
@@ -46,12 +48,13 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "### Extract the downloaded dataset\n",
     "\n",
-    "We extract all the downloaded files and only remain `val2017` subset by removing `train2017` annotation files. After running this cell, you will get the following directory structure.\n",
+    "We extract all the downloaded files and only remaining `val2017` subset by removing `train2017` annotation files. After running this cell, you will get the following directory structure.\n",
```

Comment:
```suggestion
    "We extract all the downloaded files and remove `train2017` annotation files (e.g., `instances_train2017.json`) because we only downloaded the `val2017` images. After running this cell, you will get the following directory structure.\n",
```

Reply:
Reflected in 7e11be7d13354f76c64329077ecad65d5a33368a.